services/ml/BUILD.gn

```diff
@@ -20,7 +20,7 @@ if (is_win) {
   os_folder = "Linux64"
 } else if (is_mac) {
   dnnl_dll = "libdnnl.1.1.dylib"
-  omp_dll = "libomp.dylib"
+  omp_dll = "libiomp5.dylib"
```

Comment:
Do we still need `libomp.dylib`, e.g. for debug build?

Reply:
@lisa0314 , any comments?

Reply:
@huningxin , I suppose we need `libomp.dylib` for local build.

Reply:
what do you mean "local build"? if we need it, should be keep "libomp.dylib"?

Reply:
I mean it need to depend `libomp.dylib` if I build it from source. If we just use the release version, we don't need to keep`libomp`. So I think we don't need to keep `libomp.dylib`.

Reply:
sounds good